.tekton/integration-tests/lightspeed-rag-content-image-verification.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  annotations:
+    pipelinesascode.tekton.dev/task: "[ols-installer, ols-e2e-task]"
+  name: ols-integration-tests-pipeline
+spec:
+  description: |
+    This pipeline automates the process of running end-to-end tests for OpenShift Lightspeed
+    using a ROSA (Red Hat OpenShift Service on AWS) cluster. The pipeline provisions
+    the ROSA cluster, installs the OpenShift Lightspeed operator using the installer, runs the tests, collects artifacts,
+    and finally deprovisions the ROSA cluster.
+  params:
+    - name: SNAPSHOT
+      description: 'The JSON string representing the snapshot of the application under test.'
+      default: '{"components": [{"name":"test-app", "containerImage": "quay.io/example/repo:latest"}]}'
+      type: string
+    - name: test-name
+      description: 'The name of the test corresponding to a defined Konflux integration test.'
+      default: 'ols-rag-content-verification'
+    - name: namespace
+      description: 'Namespace to run tests in'
+      default: 'openshift-lightspeed'
+  tasks:
+    - name: get-information-from-snapshot
+      description: Task to get information from snapshot
+      taskSpec:
+        results:
+          - name: rag-content-image
+            value: "$(steps.get-rag-content-image.results.rag-content-image)"
+          - name: commit
+            value: "$(steps.get-rag-content-image.results.commit)"
+        steps:
+          - name: get-rag-content-image
+            image: registry.redhat.io/openshift4/ose-cli:latest
+            env:
+              - name: SNAPSHOT
+                value: $(params.SNAPSHOT)
+            results:
+              - name: rag-content-image
+                type: string
+                description: "service image from snapshot"
+              - name: commit
+                type: string
+                description: "commit sha to be used in tests"
+            script: |
+              dnf -y install jq
+              echo -n "$(jq -r --arg component_name "lightspeed-rag-content" '.components[] | select(.name == $component_name) | .containerImage' <<< "$SNAPSHOT")" > $(step.results.rag-content-image.path)
+              echo -n "$(jq -r --arg component_name "lightspeed-rag-content" '.components[] | select(.name == $component_name) | .source.git.revision' <<< "$SNAPSHOT")" > $(step.results.commit.path)
+    - name: ols-rag-content-verification
+      description: Task to verify rag content contains expected paths
+      runAfter:
+        - get-information-from-snapshot
+      params:
+        - name: ragImage
+          value: $(tasks.get-information-from-snapshot.results.rag-content-image)
+        - name: commit
+          value: $(tasks.get-information-from-snapshot.results.commit)
+      taskSpec:
+        params:
+          - ragImage
+          - commit
```

Comment:
This `params` field is malformed. Use the upstream Tekton docs to understand which fields you need to populate: https://tekton.dev/docs/pipelines/tasks/#specifying-parameters